model/task/task.go

```diff
@@ -2522,7 +2534,9 @@ func (t *Task) Archive() error {
 	return nil
 }
 
-func ArchiveMany(tasks []Task) error {
+// Execution argument is for display tasks to override the natural incrementing
+// when archiving. To incremenent all executions, put execution = -1
+func ArchiveMany(tasks []Task, execution int) error {
```

Comment:
if we want to specify that the execution tasks have the same execution as their display task, but be incremented for all other tasks, would make sense to make a second function like ArchiveManyExecutionTasks() instead of passing in a -1 every other time?

Reply:
Would it be good to have AchiveManyOverrideExecution(tasks []Tasks, execution int) as a good description for how that function differs from the last one? I was also thinking having ArchiveMany call this with -1 and keep the logic that it has? Or separating the logic (pro more clear, con more copy/paste code)?